### PR TITLE
[90X] AlCaRecoTriggerBits updates

### PR DESCRIPTION
**Pull request by mmusich**
Greetings, in this PR:
   * allow the key replacement feature (leaving the list of attached trigger bit unchanged)
   * add a python wrapper script that allows the handling of multi-IOV changes in a single call (supports both condition DB and local sqlite file inputs)
   * refresh the configuration files in the `CondTools/HLT` folder to avoid warnings
   * add instructions to use the mult-IOV workflow

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by ggovi** [comment]
+1

File: CondTools/HLT/src/AlCaRecoTriggerBitsRcdUpdate.cc [hunk 1]
```diff
@@ -129,6 +136,34 @@ bool AlCaRecoTriggerBitsRcdUpdate::removeKeysFromMap(const std::vector<std::stri
...
+bool AlCaRecoTriggerBitsRcdUpdate::replaceKeysFromMap(const std::vector<edm::ParameterSet> &alcarecoReplace,
+						      TriggerMap &triggerMap) const
+{
+  
+  std::vector<std::pair<std::string,std::string> > keyPairs;
+  
+  for(auto &iSet : alcarecoReplace ){
+    const std::string oldKey(iSet.getParameter<std::string>("oldKey"));
+    const std::string newKey(iSet.getParameter<std::string>("newKey"));
+    keyPairs.push_back(std::make_pair(oldKey,newKey));
```
Comment (davidlange6) [inline CondTools/HLT/src/AlCaRecoTriggerBitsRcdUpdate.cc]: @mmusich - please reserve the size of keyPairs. It is known in this case.

**Comment by mmusich** [comment]
@davidlange6 done

**Pull request merged**